### DISCUSSION TITLE
記事をアップデートしたらタイムラインのトップに出して欲しいです。

### DISCUSSION THREAD

**Discussion by fygar256** [increments/qiita-discussions#915] [要望・アイデア・不具合]
例えば、このプログラムの記事のバグが取れたとか、アップデートしたとかで、記事を修正した時、タイムラインのトップに出てくるようにして欲しいです。どうも、記事をアップデートしても、タイムラインの中の記事の位置がそのままのようなので。トップへ出すかどうかはエディットの終了時に指定できたら良いです。

同様の要望が既にあればごめんなさい。よろしくお願いします。

**Comment by fygar256** [comment]
普通は、そう頻繁に記事のアップデートはないと思うので。

**Comment by shiracamus** [comment]
記事公開直後は修正したり追記したり、編集リクエストがあったり、わりと更新が多いこともあります。そのたびに上位に表示されるのはうっとおしいかもしれませんね。
編集して公開するときに 「ストックしているユーザーに変更の通知を送る」をチェックしておくと、ストックしてる人には通知が行きます。必要な人だけに届けられて、タイムラインも上がった気がするのですが、これでは不十分でしょうか？

**Reply by radian-jp** [reply]
私も同意見ですね。投稿した後に細かい表現を書き直したくなる事は結構ありますし、その都度上に上がってこられるのは、本当の新着記事を見たい人にとってはストレスになると思います。
大幅に書き直しをしたとかであれば、改訂版の記事を別に投稿するとかでよいのではないでしょうか。

**Comment by tanaka-asami415** [comment]
@fygar256 
リクエストありがとうございます。

ユーザーさんの投稿スタイルによっては、何度も更新される方もいるため、その都度タイムラインに表示されるのは、shiracamus さんや radian-jp さんにご記載いただいたような懸念点があります。

同様のリクエストとして、「投稿時にタイムラインに出て欲しい」というご意見をいただくことがあるのですが、こちらは検討していきたいと考えています🙇

**Reply by fygar256** [reply]
そうなんですね。知りませんでした。わかりましたです。

**Comment by fygar256** [comment]
close
